setup.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: default
+---
+
+# Course setup
+
+To get started, you should have the following three elements setup:
+
+1. Install Python and the required Python packages
+2. Download the course material to your computer
+3. Start Jupyter notebook
+
+In the following sections, more details are provided for each of these steps. When all three are done, you are ready to start coding!
+
+
+## 1. Install Python and the required Python packages
+
+For scientific and data analysis, we recommend to use Anaconda (or Miniconda) (https://www.continuum.io/downloads), which provide a Python distribution that includes the scientific libraries (this recommendation applies to all platforms, so for both Window, Linux and Mac), instead of installing Python as such. After installation, proceed with the setup.
```

Comment:
I am testing out the jekyll site locally, and it seems I need to put angle brackets around the urls to have them working: `<..url..>`

Reply:
mmh, I was not aware of it, I tested the merged version, links seem to work fine!